### ISSUE TITLE
Consider forking Wavebox Classic

### ISSUE THREAD

**Issue by MichaelTunnell** [openWMail/openWMail#80]
Wavebox Classic was still open source but apparently they have abandoned open source now that they have reached critical mass so it might be valuable to make OpenWave. :laughing:

**Comment by geekgonecrazy** [comment]
That’s a shame 😔 though not that surprising.

Google kinda crippled our fork here and neither of us that forked this have had time to fix it.

Also personally started self hosting my mail.  So even less motivation for me 🙈

**Comment by MichaelTunnell** [comment]
that's why I suggest forking Wavebox because it is vastly different than WMail was and Google is kind of irrelevant to the value of the project. I dont care about the GMail support in regards to how I was using Wavebox after a while.

**Comment by geekgonecrazy** [comment]
Yeah makes sense.  Maybe if we get interest to take up and help Maintain ( #83 ) then they would take that up.
